### DISCUSSION TITLE
ALGO/USD price showing

### DISCUSSION THREAD

**Discussion by marucat27** [asastats/channel#255] [Ideas]
I'm wondering if it would make sense to show the current price of ALGO/USD on the ASA STATs page?  It could be at the top corner possibly?  Or would that disrupt the cleanliness of the existing UI?  Thoughts?

**Comment by kerrilija** [comment]
When you click on your total algo value, the price shows up.

**Reply by marucat27** [reply]
Thanks!  Learning new things of ASA Stats daily 😬
